### DISCUSSION TITLE
Re-scan OCR input?

### DISCUSSION THREAD

**Discussion by hereinmyhead** [jonaswinkler/paperless-ng#1329] [Support]
I have imported about 2K+ documents and I been spending quite of time on orginazing with metadata. Then, today, I realized that I haven't config the OCR scanner to my native language and therefore the OCR metadata misses some letters.

I have been seachring i the documentation but I can't find any answer on my question:

Is there any way force Paperless-ng to rescan OCR on my document without loss of taggning etc?

Thank you!

**Comment by Pyromane** [comment]
Unfortunately not yet, see #809
